6/모의고사.js

```diff
@@ -0,0 +1,19 @@
+function solution(answers) {
+  const mathHater1 = [1, 2, 3, 4, 5];
+  const mathHater2 = [2, 1, 2, 3, 2, 4, 2, 5];
+  const mathHater3 = [3, 3, 1, 1, 2, 2, 4, 4, 5, 5];
+  const ranks = [];
+  const scores = [0, 0, 0];
+
+  for (let a in answers) {
+    if (mathHater1[a % mathHater1.length] === answers[a]) scores[0] += 1;
+    if (mathHater2[a % mathHater2.length] === answers[a]) scores[1] += 1;
+    if (mathHater3[a % mathHater3.length] === answers[a]) scores[2] += 1;
+  }
```

Comment:
```suggestion
  for (let a in answers) {
    if (mathHater1[a % mathHater1.length] === answers[a]) scores[0] += 1;
    if (mathHater2[a % mathHater2.length] === answers[a]) scores[1] += 1;
    if (mathHater3[a % mathHater3.length] === answers[a]) scores[2] += 1;
  }
```
이 부분은 반복문을 사용하지 않고 filter()메서드를 이용해서도 구현할 수 있어요!! 한번 해보세요.
그리고 a변수 말고 'answer'이런 식으로 변수명에 조금 더 신경 써보아요~